docs/First-Installation.md

```diff
@@ -109,53 +109,36 @@ The Hotkey button enables you to press it in combination with another button to
-
-Software can be installed from the RetroPie-Setup script - which is accessible from the RetroPie menu on EmulationStation. Once there you can navigate to "Manage Packages" where you will see various sections. In each section are lists of packages that can be installed (and it will show what is currently installed). Stable additional packages are under the "Optional" section, with more unstable packages listed under experimental. The packages are ordered first by type (emulators / libretro cores / ports), then alphabetically. By selecting a package you can choose to install it, or remove it. Some packages also have additional configurations.
-
 ## Transferring ROMs
 
-You will not see any game systems (NES, n64, Playstation, etc) on the main menu until you add ROMs! Visit the [Transferring ROMs](Transferring-Roms) page to learn how to transfer ROMs to RetroPie.
+You will not see any game systems (NES, n64, Playstation, etc) on the system list until you add ROMs! Visit the [Transferring ROMs](Transferring-Roms) page to learn how to transfer ROMs to RetroPie.
 
-## AUDIO
+## Audio
 
-In general RetroPie audio will work out of the box without any tweaking, but if you have audio issues you should follow the instructions on the [Sound Issues Page](Sound-Issues) to fix them. You will most likely need to visit the [Sound Issues Page](Sound-Issues) if you are using a USB Audio device, or if you are using an aftermarket RPi HAT add-on audio device (such as a Justboom sound card).
+In general RetroPie audio will work out-of-the-box without any tweaking if using HDMI, but if you have audio issues you should follow the instructions on the [Sound Issues Page](Sound-Issues) to fix them. You will most likely need to visit this page if you are using a USB audio device, or if you are using an aftermarket RPi HAT add-on audio device (such as a Justboom sound card).
 
 ## PLAY!
 
-After you've added your roms you need to restart EmulationStation in order for them to show up. You can restart EmulationStation from the start menu, or by rebooting your pi with `sudo reboot`. 
+After you've transferred your ROMs, you need to restart EmulationStation in order for them to show up. You can restart EmulationStation by pressing Start > Quit > Restart EmulationStation, or with SSH access by rebooting your pi with `sudo reboot`. Once rebooted, you should see the game systems appear on the system list.
+
+## Additional Setup Options
+
+* [Configure Wifi](Wifi)
+* [Enable SSH](SSH)
+* Configure more controllers. This can be done after plugging in the new controller and pressing *Start* on your controller and selecting *Configure Input*
+* [Install additional emulators/cores](https://retropie.org.uk/docs/Updating-RetroPie/#updatinginstalling-individual-packages)
```

Comment:
I feel like we're losing something deleting the intro to this:

> On RetroPie 4.0+, not everything is installed by default. The pre-made images contain the best working emulators for each system supported by the hardware. This should cover everything most users would be doing. Ports like quake and doom and some other emulators like ScummVM can be installed later.

But I would streamline it a bit to something like

> In RetroPie, not everything is installed by default. The pre-made images contain the best-working emulators for each system supported by the hardware. This should cover typical use, but if you want to install additional emulators or ports, [follow these instructions](https://retropie.org.uk/docs/Updating-RetroPie/#updatinginstalling-individual-packages)).

Reply:
Reworded your example a bit. How's that?

Reply:
\*poke*

Reply:
That's fine however I suppose technically it should be 

> The pre-made images contain the best-working emulators for the more common systems supported by the hardware.

some well best-working emulators aren't installed simply because they're more niche systems/ports and would be taking up space for most users.

Reply:
Done. Do you prefer to resolve these conversations or should I?